packages/grid/_modules_/grid/locales/ptBR.ts

```diff
@@ -6,6 +6,7 @@ const ptBRGrid: Partial<GridLocaleText> = {
   // Root
   rootGridLabel: 'Grade',
   noRowsLabel: 'Nenhuma linha',
+  allRowsFilteredLabel: 'Todos os registros estão filtrados',
```

Comment:
It's a side note, but I was thinking if we should keep the translations up to date somehow. Anyone any ideas about this? @oliviertassinari are there efforts like this in the core?

Reply:
The situation is different in the core as we have fewer translations key, and change them less frequently.

However, what we have done is to update all the locales with the commented keys in English. It makes a clear signal for the community that it's not handled and up for grab.

Reply:
It would be interesting to write a script that automatically detect out of date locale and transform them. It's typically the type of infrastructure work that Sebastian has been doing in the core.